packages/sdk-react/src/mocks/payables/payableHandlers.ts

```diff
@@ -168,6 +168,22 @@ export const payableHandlers = [
 
       const url = new URL(request.url);
       const limit = Number(url.searchParams.get('limit') ?? '10');
+      const id__in = url.searchParams.getAll('id__in');
+
+      if (id__in.length > 0) {
+        const payables = payableFixturePages
+          .slice(0, id__in.length)
```

Comment:
I think .slice() here is redundant, since map inherently limits the iteration to the length of id__in.

```ts
  const payables = id__in.map((id, index) => ({
    ...payableFixturePages[index],
    id: id,
  }));
```

Reply:
thanks! fixed